Crate does not compile on 1.56

```bash
error[E0557]: feature has been removed
   --> /home/azureuser/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd_2-0.3.5/src/lib.rs:215:5
    |
215 |     const_generics,
    |     ^^^^^^^^^^^^^^ feature has been removed
```

> cargo 1.56.0-nightly (f559c109c 2021-08-26)

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -212,7 +212,7 @@
 //! guide](https://rust-lang-nursery.github.io/packed_simd/perf-guide/)
 
 #![feature(
-    const_generics,
+    adt_const_params,
     repr_simd,
     rustc_attrs,
     platform_intrinsics,

